### DISCUSSION TITLE
Add CCOpt.{none,some}?

### DISCUSSION THREAD

**Discussion by favonia** [c-cube/ocaml-containers#382] [Ideas]
I have been wondering what design principles might apply regarding these functions from `Stdlib.Option`, as I seem to be able to argue from the both sides whether these trivial functions should be included in `containers`. On the one hand, these functions do not suffer from various legacy issues (throwing exceptions instead of returning `result`, not tail-recursive, etc.), and this would make migration from the standard library easier.  On the other hand, one can simply write `None` for `none` and `pure` for `some`.

**Comment by c-cube** [comment]
I haven't followed closely, did the stdlib already add `some`/`none`? If so, we can add them, they really don't cost anything :)

**Reply by favonia** [reply]
Yes the standard library already have them.

**Comment by c-cube** [comment]
done in 25660ee2c19c7089c7a1beff8f9ea9f611b283b9
